### YamlMime:ManagedReference
items:
- uid: System.NotFiniteNumberException
  id: NotFiniteNumberException
  children:
  - System.NotFiniteNumberException.#ctor
  - System.NotFiniteNumberException.#ctor(System.Double)
  - System.NotFiniteNumberException.#ctor(System.String)
  - System.NotFiniteNumberException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.NotFiniteNumberException.#ctor(System.String,System.Double)
  - System.NotFiniteNumberException.#ctor(System.String,System.Exception)
  - System.NotFiniteNumberException.#ctor(System.String,System.Double,System.Exception)
  - System.NotFiniteNumberException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.NotFiniteNumberException.OffendingNumber
  langs:
  - csharp
  name: NotFiniteNumberException
  nameWithType: NotFiniteNumberException
  fullName: System.NotFiniteNumberException
  type: Class
  summary: "De uitzondering die wordt gegenereerd wanneer een drijvende-kommawaarde positieve oneindigheid, negatief oneindig of geen getal (NaN is)."
  remarks: "NotFiniteNumberException is beschikbaar voor programmeertalen die de concepten van oneindig en geen getal in drijvende-bewerkingen niet ondersteunen.       NotFiniteNumberException maakt gebruik van de HRESULT COR_E_NOTFINITENUMBER, dat de waarde 0x80131528 heeft.       Zie voor een lijst met eerste eigenschapswaarden voor een exemplaar van NotFiniteNumberException de <xref:System.NotFiniteNumberException.%23ctor%2A>constructors.</xref:System.NotFiniteNumberException.%23ctor%2A>"
  syntax:
    content: >-
      [System.Runtime.InteropServices.ComVisible(true)]

      public class NotFiniteNumberException : ArithmeticException
  inheritance:
  - System.ArithmeticException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.NotFiniteNumberException.#ctor
  id: '#ctor'
  parent: System.NotFiniteNumberException
  langs:
  - csharp
  name: NotFiniteNumberException()
  nameWithType: NotFiniteNumberException.NotFiniteNumberException()
  fullName: System.NotFiniteNumberException.NotFiniteNumberException()
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.NotFiniteNumberException&quot;> </xref> klasse."
  remarks: "Deze constructor initialiseert de <xref:System.Exception.Message%2A>eigenschap van het nieuwe exemplaar op een systeem geleverd bericht die de fout wordt beschreven, zoals &quot;de aangetroffen aantal is geen eindige hoeveelheid.&quot;</xref:System.Exception.Message%2A> Dit bericht wordt rekening gehouden met de huidige systeemcultuur.       De volgende tabel toont de eerste waarden voor een exemplaar van <xref:System.NotFiniteNumberException>.</xref:System.NotFiniteNumberException>      | Eigenschap | Waarde |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Een null-verwijzing (`Nothing` in Visual Basic). |   | <xref:System.Exception.Message%2A>| De gelokaliseerde fouttekenreeks bericht. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public NotFiniteNumberException ();
    parameters: []
  overload: System.NotFiniteNumberException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.NotFiniteNumberException.#ctor(System.Double)
  id: '#ctor(System.Double)'
  parent: System.NotFiniteNumberException
  langs:
  - csharp
  name: NotFiniteNumberException(Double)
  nameWithType: NotFiniteNumberException.NotFiniteNumberException(Double)
  fullName: System.NotFiniteNumberException.NotFiniteNumberException(Double)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.NotFiniteNumberException&quot;> </xref> klasse met het ongeldige getal."
  remarks: "De `offendingNumber` parameter moet zowel een <xref:System.Double>en een ongeldig getal.</xref:System.Double>       De volgende tabel toont de eerste waarden voor een exemplaar van <xref:System.NotFiniteNumberException>.</xref:System.NotFiniteNumberException>      | Eigenschap | Waarde |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Een null-verwijzing (`Nothing` in Visual Basic). |   | <xref:System.NotFiniteNumberException.OffendingNumber%2A>| Ongeldig nummer. |</xref:System.NotFiniteNumberException.OffendingNumber%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public NotFiniteNumberException (double offendingNumber);
    parameters:
    - id: offendingNumber
      type: System.Double
      description: "De waarde van het argument dat de uitzondering heeft veroorzaakt."
  overload: System.NotFiniteNumberException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.NotFiniteNumberException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.NotFiniteNumberException
  langs:
  - csharp
  name: NotFiniteNumberException(String)
  nameWithType: NotFiniteNumberException.NotFiniteNumberException(String)
  fullName: System.NotFiniteNumberException.NotFiniteNumberException(String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.NotFiniteNumberException&quot;> </xref> klasse met een opgegeven foutbericht."
  remarks: "Zorg ervoor dat de `message` tekenreeks is onbegrijpelijk voor de gebruiker en dat deze is geconfigureerd voor de huidige cultuur.       De volgende tabel toont de eerste waarden voor een exemplaar van <xref:System.NotFiniteNumberException>.</xref:System.NotFiniteNumberException>      | Eigenschap | Waarde |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Een null-verwijzing (`Nothing` in Visual Basic). |   | <xref:System.Exception.Message%2A>| De fouttekenreeks bericht. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public NotFiniteNumberException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Het bericht dat de fout wordt beschreven."
  overload: System.NotFiniteNumberException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.NotFiniteNumberException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.NotFiniteNumberException
  langs:
  - csharp
  name: NotFiniteNumberException(SerializationInfo,StreamingContext)
  nameWithType: NotFiniteNumberException.NotFiniteNumberException(SerializationInfo,StreamingContext)
  fullName: System.NotFiniteNumberException.NotFiniteNumberException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.NotFiniteNumberException&quot;> </xref> klasse met de geserialiseerde gegevens."
  remarks: "Deze constructor is aangeroepen tijdens de deserialisatie op het object exception verzonden via een stroom opnieuw samenstellen."
  syntax:
    content: protected NotFiniteNumberException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Het object dat de gegevens geserialiseerd object bevat."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "De contextuele informatie over de bron- of doelserver."
  overload: System.NotFiniteNumberException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.NotFiniteNumberException.#ctor(System.String,System.Double)
  id: '#ctor(System.String,System.Double)'
  parent: System.NotFiniteNumberException
  langs:
  - csharp
  name: NotFiniteNumberException(String,Double)
  nameWithType: NotFiniteNumberException.NotFiniteNumberException(String,Double)
  fullName: System.NotFiniteNumberException.NotFiniteNumberException(String,Double)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.NotFiniteNumberException&quot;> </xref> klasse met een opgegeven foutbericht en ongeldig nummer."
  remarks: "Zorg ervoor dat de `message` tekenreeks is onbegrijpelijk voor de gebruiker en dat deze is geconfigureerd voor de huidige cultuur.       De volgende tabel toont de eerste waarden voor een exemplaar van <xref:System.NotFiniteNumberException>.</xref:System.NotFiniteNumberException>      | Eigenschap | Waarde |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Een null-verwijzing (`Nothing` in Visual Basic). |   | <xref:System.Exception.Message%2A>| De fouttekenreeks bericht. |   | <xref:System.NotFiniteNumberException.OffendingNumber%2A>| Ongeldig nummer. |</xref:System.NotFiniteNumberException.OffendingNumber%2A> </xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public NotFiniteNumberException (string message, double offendingNumber);
    parameters:
    - id: message
      type: System.String
      description: "Het bericht dat de fout wordt beschreven."
    - id: offendingNumber
      type: System.Double
      description: "De waarde van het argument dat de uitzondering heeft veroorzaakt."
  overload: System.NotFiniteNumberException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.NotFiniteNumberException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.NotFiniteNumberException
  langs:
  - csharp
  name: NotFiniteNumberException(String,Exception)
  nameWithType: NotFiniteNumberException.NotFiniteNumberException(String,Exception)
  fullName: System.NotFiniteNumberException.NotFiniteNumberException(String,Exception)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.NotFiniteNumberException&quot;> </xref> klasse met een opgegeven foutbericht en een verwijzing naar de interne uitzondering die de hoofdoorzaak van deze uitzondering."
  remarks: "Een uitzondering die is gegenereerd als direct resultaat van een eerdere uitzondering kan bevatten een verwijzing naar de vorige uitzondering in de <xref:System.Exception.InnerException%2A>eigenschap.</xref:System.Exception.InnerException%2A> De <xref:System.Exception.InnerException%2A>eigenschap dezelfde waarde die is doorgegeven aan de constructor of `null` als de <xref:System.Exception.InnerException%2A>eigenschap levert niet de waarde van de binnenste uitzondering voor de constructor.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       De volgende tabel toont de eerste waarden voor een exemplaar van <xref:System.NotFiniteNumberException>.</xref:System.NotFiniteNumberException>      | Eigenschap | Waarde |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| De verwijzing naar de interne uitzondering. |   | <xref:System.Exception.Message%2A>| De fouttekenreeks bericht. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public NotFiniteNumberException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "Het foutbericht met uitleg over de reden voor de uitzondering."
    - id: innerException
      type: System.Exception
      description: "De uitzondering die de oorzaak van de huidige uitzondering. Als de `innerException` parameter is niet een null-verwijzing (<xref uid=&quot;langword_csharp_Nothing&quot; name=&quot;Nothing&quot; href=&quot;&quot;> </xref> in Visual Basic), de huidige uitzondering is opgetreden tijdens een <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> blok die verantwoordelijk is voor de interne uitzondering."
  overload: System.NotFiniteNumberException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.NotFiniteNumberException.#ctor(System.String,System.Double,System.Exception)
  id: '#ctor(System.String,System.Double,System.Exception)'
  parent: System.NotFiniteNumberException
  langs:
  - csharp
  name: NotFiniteNumberException(String,Double,Exception)
  nameWithType: NotFiniteNumberException.NotFiniteNumberException(String,Double,Exception)
  fullName: System.NotFiniteNumberException.NotFiniteNumberException(String,Double,Exception)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.NotFiniteNumberException&quot;> </xref> klasse met een opgegeven foutbericht, ongeldig getal en een verwijzing naar de interne uitzondering die de hoofdoorzaak van deze uitzondering."
  remarks: "Een uitzondering die is gegenereerd als direct resultaat van een eerdere uitzondering kan bevatten een verwijzing naar de vorige uitzondering in de <xref:System.Exception.InnerException%2A>eigenschap.</xref:System.Exception.InnerException%2A> De <xref:System.Exception.InnerException%2A>eigenschap dezelfde waarde die is doorgegeven aan de constructor of `null` als de <xref:System.Exception.InnerException%2A>eigenschap levert niet de waarde van de binnenste uitzondering voor de constructor.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       De volgende tabel toont de eerste waarden voor een exemplaar van <xref:System.NotFiniteNumberException>.</xref:System.NotFiniteNumberException>      | Eigenschap | Waarde |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| De verwijzing naar de interne uitzondering. |   | <xref:System.Exception.Message%2A>| De fouttekenreeks bericht. |   | <xref:System.NotFiniteNumberException.OffendingNumber%2A>| Ongeldig nummer. |</xref:System.NotFiniteNumberException.OffendingNumber%2A> </xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public NotFiniteNumberException (string message, double offendingNumber, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "Het foutbericht met uitleg over de reden voor de uitzondering."
    - id: offendingNumber
      type: System.Double
      description: "De waarde van het argument dat de uitzondering heeft veroorzaakt."
    - id: innerException
      type: System.Exception
      description: "De uitzondering die de oorzaak van de huidige uitzondering. Als de `innerException` parameter is niet een null-verwijzing (<xref uid=&quot;langword_csharp_Nothing&quot; name=&quot;Nothing&quot; href=&quot;&quot;> </xref> in Visual Basic), de huidige uitzondering is opgetreden tijdens een <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> blok die verantwoordelijk is voor de interne uitzondering."
  overload: System.NotFiniteNumberException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.NotFiniteNumberException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.NotFiniteNumberException
  langs:
  - csharp
  name: GetObjectData(SerializationInfo,StreamingContext)
  nameWithType: NotFiniteNumberException.GetObjectData(SerializationInfo,StreamingContext)
  fullName: System.NotFiniteNumberException.GetObjectData(SerializationInfo,StreamingContext)
  type: Method
  assemblies:
  - mscorlib
  namespace: System
  summary: "Stelt de <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> object met het ongeldige aantal en de aanvullende uitzonderingsgegevens."
  remarks: "<xref:System.TypeLoadException.GetObjectData%2A>Hiermee stelt u een <xref:System.Runtime.Serialization.SerializationInfo>met alle uitzondering objectgegevens die zijn gericht voor serialisatie.</xref:System.Runtime.Serialization.SerializationInfo></xref:System.TypeLoadException.GetObjectData%2A> Tijdens de deserialisatie-de exception-object is aangemaakt de <xref:System.Runtime.Serialization.SerializationInfo>verzonden via de stroom.</xref:System.Runtime.Serialization.SerializationInfo>       Zie voor meer informatie <xref:System.Runtime.Serialization.SerializationInfo>.</xref:System.Runtime.Serialization.SerializationInfo>"
  syntax:
    content: public override void GetObjectData (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Het object dat de gegevens geserialiseerd object bevat."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "De contextuele informatie over de bron- of doelserver."
  overload: System.NotFiniteNumberException.GetObjectData*
  exceptions:
  - type: System.ArgumentNullException
    commentId: T:System.ArgumentNullException
    description: "The <code>info</code> object is <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;></xref>."
  platform:
  - net462
- uid: System.NotFiniteNumberException.OffendingNumber
  id: OffendingNumber
  parent: System.NotFiniteNumberException
  langs:
  - csharp
  name: OffendingNumber
  nameWithType: NotFiniteNumberException.OffendingNumber
  fullName: System.NotFiniteNumberException.OffendingNumber
  type: Property
  assemblies:
  - mscorlib
  namespace: System
  summary: "Hiermee haalt u de ongeldig getal dat een positieve oneindigheid, een negatief oneindig of geen getal (NaN)."
  remarks: "De eigenschap OffendingNumber retourneert dezelfde waarde als is doorgegeven aan de constructor, of nul als geen waarde is opgegeven. De eigenschap OffendingNumber voor het aanpassen van de inhoud of de indeling van de ongeldige waarde te vervangen.       Deze eigenschap is alleen-lezen."
  syntax:
    content: public double OffendingNumber { get; }
    return:
      type: System.Double
      description: "De ongeldig getal."
  overload: System.NotFiniteNumberException.OffendingNumber*
  exceptions: []
  platform:
  - net462
references:
- uid: System.ArithmeticException
  isExternal: false
  name: System.ArithmeticException
- uid: System.ArgumentNullException
  isExternal: true
  name: System.ArgumentNullException
- uid: System.NotFiniteNumberException.#ctor
  parent: System.NotFiniteNumberException
  isExternal: false
  name: NotFiniteNumberException()
  nameWithType: NotFiniteNumberException.NotFiniteNumberException()
  fullName: System.NotFiniteNumberException.NotFiniteNumberException()
- uid: System.NotFiniteNumberException.#ctor(System.Double)
  parent: System.NotFiniteNumberException
  isExternal: false
  name: NotFiniteNumberException(Double)
  nameWithType: NotFiniteNumberException.NotFiniteNumberException(Double)
  fullName: System.NotFiniteNumberException.NotFiniteNumberException(Double)
- uid: System.Double
  parent: System
  isExternal: true
  name: Double
  nameWithType: Double
  fullName: System.Double
- uid: System.NotFiniteNumberException.#ctor(System.String)
  parent: System.NotFiniteNumberException
  isExternal: false
  name: NotFiniteNumberException(String)
  nameWithType: NotFiniteNumberException.NotFiniteNumberException(String)
  fullName: System.NotFiniteNumberException.NotFiniteNumberException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.NotFiniteNumberException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.NotFiniteNumberException
  isExternal: false
  name: NotFiniteNumberException(SerializationInfo,StreamingContext)
  nameWithType: NotFiniteNumberException.NotFiniteNumberException(SerializationInfo,StreamingContext)
  fullName: System.NotFiniteNumberException.NotFiniteNumberException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.NotFiniteNumberException.#ctor(System.String,System.Double)
  parent: System.NotFiniteNumberException
  isExternal: false
  name: NotFiniteNumberException(String,Double)
  nameWithType: NotFiniteNumberException.NotFiniteNumberException(String,Double)
  fullName: System.NotFiniteNumberException.NotFiniteNumberException(String,Double)
- uid: System.NotFiniteNumberException.#ctor(System.String,System.Exception)
  parent: System.NotFiniteNumberException
  isExternal: false
  name: NotFiniteNumberException(String,Exception)
  nameWithType: NotFiniteNumberException.NotFiniteNumberException(String,Exception)
  fullName: System.NotFiniteNumberException.NotFiniteNumberException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.NotFiniteNumberException.#ctor(System.String,System.Double,System.Exception)
  parent: System.NotFiniteNumberException
  isExternal: false
  name: NotFiniteNumberException(String,Double,Exception)
  nameWithType: NotFiniteNumberException.NotFiniteNumberException(String,Double,Exception)
  fullName: System.NotFiniteNumberException.NotFiniteNumberException(String,Double,Exception)
- uid: System.NotFiniteNumberException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.NotFiniteNumberException
  isExternal: false
  name: GetObjectData(SerializationInfo,StreamingContext)
  nameWithType: NotFiniteNumberException.GetObjectData(SerializationInfo,StreamingContext)
  fullName: System.NotFiniteNumberException.GetObjectData(SerializationInfo,StreamingContext)
- uid: System.NotFiniteNumberException.OffendingNumber
  parent: System.NotFiniteNumberException
  isExternal: false
  name: OffendingNumber
  nameWithType: NotFiniteNumberException.OffendingNumber
  fullName: System.NotFiniteNumberException.OffendingNumber
- uid: System.NotFiniteNumberException.#ctor*
  parent: System.NotFiniteNumberException
  isExternal: false
  name: NotFiniteNumberException
  nameWithType: NotFiniteNumberException.NotFiniteNumberException
- uid: System.NotFiniteNumberException.GetObjectData*
  parent: System.NotFiniteNumberException
  isExternal: false
  name: GetObjectData
  nameWithType: NotFiniteNumberException.GetObjectData
- uid: System.NotFiniteNumberException.OffendingNumber*
  parent: System.NotFiniteNumberException
  isExternal: false
  name: OffendingNumber
  nameWithType: NotFiniteNumberException.OffendingNumber
